Phonebook.Frontend/src/app/pages/dashboard/components/bookmarked/bookmarked.component.html

```diff
@@ -0,0 +1,82 @@
+<div class="pb-bookmarked">
+  <div class="pb-bookmarked-title">
+    <div class="different-cards">
+      <button class="navigation-button" mat-button [routerLink]="['/dashboard/bookmarked']">
+        <div class="card-navigation">
+          <mat-icon>bookmark</mat-icon>
+          <span class="navigation-name" i18n="DashboardComponent|Title of the Bookmarked people section@@DashboardComponentTitleBookmark">
+            Bookmarked People
+          </span>  
+        </div>
+        </button>
+      </div>
+    <mat-form-field>
+      <mat-select [(value)]="favoriteSort" (valueChange)="changeOrder()">
+        <mat-option
+          value=""
+          i18n="
+             DashboardComponent|The standard (customizable) order of the favorite cards@@DashboardComponentOrderNone
+
+          "
+        >
+          Custom Order</mat-option
+        >
+        <mat-option
+          value="asc"
+          i18n="DashboardComponent|The alphabetical asc order of the favorite cards@@DashboardComponentOrderAsc"
+        >
+          Alphabetical asc</mat-option
+        >
+        <mat-option
+          value="desc"
+          i18n="DashboardComponent|The alphabetical desc order of the favorite cards@@DashboardComponentOrderDesc"
+        >
+          Alphabetical desc</mat-option
+        >
+      </mat-select>
+    </mat-form-field>
+  </div>
+  <!-- This is a workaround for https://github.com/angular/material2/issues/13372 -->
+  <div cdkDropListGroup class="pb-bookmarks-list pb-flex-row  pb-overflow-auto">
+    <div
+      *ngFor="let person of bookmarkedPersons; index as i"
+      cdkDropList
+      [cdkDropListData]="i"
+      cdkDropListOrientation="horizontal"
+      class="card-container"
+    >
+      <app-user-small-card
+        [person]="person"
+        [actionButtonIcon]="'close'"
+        [actionButtonClasses]="'white'"
+        (actionButtonClicked)="removeFromBookmarkedPersons(person)"
+        class="pb-small-card pb-card"
+        cdkDrag
+        (cdkDragEntered)="entered($event)"
+        (cdkDragEnded)="ended($event)"
+        [cdkDragData]="i"
+        [cdkDragDisabled]="favoriteSort != ''"
+      ></app-user-small-card>
+    </div>
+    <div
+    *ngIf="bookmarkedPersons.length === 0"
+    i18n="DashboardComponent|Message displayed if no people are bookmarked@@DashboardComponentDescriptionBookmark"
+  >
+    You haven't bookmarked anybody yet, look for the button
+    <button mat-icon-button>
+      <mat-icon>bookmark_border</mat-icon>
+    </button>
+    on a workmates page.
+  </div>
+  </div>
+  <div class="different-cards bottom-button" *ngIf='currentUser != null'>
```

Comment:
Please move the navigation to the dashboard component. 
Why? This way we can reuse the bookmarked component in other parts of the application.